### PR TITLE
Issue 38 - Scales create NamedBlobImage with wrong contentType field

### LINKED ISSUE #38: Scales create NamedBlobImage with wrong contentType field
`NamedBlobImage` `contentType` field [should be **string** not **unicode**](https://github.com/plone/plone.namedfile/blob/master/plone/namedfile/usage.rst).

But when the image is scaled, [it becomes **unicode**](https://github.com/plone/plone.namedfile/blob/master/plone/namedfile/scaling.py#L236-L241), what creates validation issues in image fields [1](https://community.plone.org/t/migration-collective-jsonify-news-item-images-lead-image-wrong-contained-type/1544), [2](https://github.com/collective/collective.cover/issues/637).

This bug was introduced in [this refactoring](https://github.com/plone/plone.namedfile/commit/33df023d09c3d18df14f7f93feeca2ba5bf19c88#diff-03f29bbadcdf640341c9061de5a07dccR299).

### PR DESCRIPTION
**Pull request by rodfersou**
closes #38

### CODE REVIEW

File: plone/namedfile/tests/test_scaling.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def testCreateScale(self):
         foo = self.scaling.scale('image', width=100, height=80)
         self.assertTrue(foo.uid)
         self.assertEqual(foo.mimetype, 'image/jpeg')
+        self.assertEqual(foo.data.contentType, 'image/jpeg')
```
Comment (hvelarde) [inline plone/namedfile/tests/test_scaling.py]: these tests need to be enhances as:

```python
>>> u'image/jpeg' == 'image/jpeg'
True
```

we need to guarantee it's an instance of `str`

**Comment by rodfersou** [comment]
Needs to figure out how to write the upgrade step.

Comment from @hvelarde 
> @rodfersou keep it simple: search for images and scales only; my only concern is about the upgrade step memory consumption.

I think all scales will provide an specific interface, and it will be searchable through **portal_catalog**.
No need to search for images, the bug happen just on scales.

@hvelarde can you validate if this is the right approach?

**Comment by rodfersou** [comment]
Or the right approach is to **search for all images**, and then look for **all scales provided in the current portal**, scale all images, and fix the scales.

**Comment by rodfersou** [comment]
I think anyone of these approaches will not fix for example the Plone 5 logo configlet, maybe it is good to check if the validation bug happen there.

**Comment by hvelarde** [comment]
we can't fix all imaginable scenarios; talk to @thet on the best approach.

we need to merge this ASAP.

**Comment by rodfersou** [comment]
@hvelarde @thet said [he can help just next week](https://github.com/plone/plone.namedfile/issues/38#issuecomment-270204820)

**Comment by rodfersou** [comment]
@thet could you please guide me on the best approach here?

**Comment by hvelarde** [comment]
@thet can you help here?

**Comment by thet** [comment]
LGTM. maybe the upgrade step is not needed at all and the contentType is just corrected the next time an image is saved. I'm merging.

@hvelarde can you report back, if this issue also needs an upgrade step?

**Pull request merged**